api/src/LinkyLink/Startup.cs

```diff
@@ -85,6 +80,9 @@ public void ConfigureServices(IServiceCollection services)
                 var xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
                 c.IncludeXmlComments(xmlPath);
             });
+            
+            // Add Application Insights Telemetry
```

Comment:
I have a PR which adds similar code. I would prefer to have this line commented out. User can uncomment the line to enable application insights if they wish to use it.

Reply:
Is there a reason why we just don't want it on all the time?

Reply:
Almost everything will be wired up with App insights but I was thinking may be attendees should enable this if they choose to go with App Insights. Anyway, I dont have a strong preference.

Reply:
I prefer leaving it there. Not really our intention to show you how to add App Insights to your app right? It's more like use it answer your business problems, or if you don't like it hook up something else. 

Go for it, swap in FluentD and send logs into Grafana.

Reply:
Let's discuss after Stand Up.  If we believe it should be off on the backend, then we need to do the same for the frontend as it doesn't make sense to collect telemetry by default on the frontend and not the backend, especially with distributed tracing enabled by default.

I personally believe that it should be on by default.  It goes back to the "who would deploy an app to production with no monitoring?".  The narrative states that we were in a hurry to go live but we at least had "some" monitoring but it was not finely tuned.

Reply:
I agree with both of you. Lets keep it on by default. I`ll change my other PR to keep it on. as well.